app/parse.py

```diff
@@ -15,8 +33,110 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def init_driver(headless: bool = False) -> webdriver.Chrome:
+    options = Options()
+    options.headless = headless
+    return webdriver.Chrome(options=options)
+
+
+def accept_cookies(driver: webdriver.Chrome) -> None:
+    try:
+        cookie = WebDriverWait(driver, 10).until(
+            ec.element_to_be_clickable((By.CLASS_NAME, "acceptCookies"))
+        )
+        cookie.click()
+    except (ElementClickInterceptedException, TimeoutException) as e:
+        print(f"Error accepting cookies: {e}")
+
+
+def click_load_more(driver: webdriver.Chrome) -> None:
+    try:
+        while True:
+            button = WebDriverWait(driver, 10).until(
+                ec.element_to_be_clickable(
+                    (By.CLASS_NAME, "ecomerce-items-scroll-more")
+                )
+            )
+            button.click()
+            time.sleep(0.5)
+    except (TimeoutException, ElementClickInterceptedException):
+        print("No more products to load or button not clickable.")
+
+
+def get_product(product_soup: BeautifulSoup) -> Product | None:
```

Comment:
The use of the union operator '|' for type hinting (e.g., `Product | None`) is only available in Python 3.10 and later. If you need compatibility with earlier versions, consider using `Optional[Product]` from the `typing` module.